x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -43,6 +43,15 @@ const createNewTrustedAppForOsScheme = <O extends OperatingSystem, F extends Con
   schema.object({
     name: schema.string({ minLength: 1, maxLength: 256 }),
     description: schema.maybe(schema.string({ minLength: 0, maxLength: 256, defaultValue: '' })),
+    effectScope: schema.oneOf([
```

Comment:
we likely will need to revisit this because I don't think using `onOf()` will work. Am I correct in that a trusted app could be assigned to to the global list **AND** to specific policies. the `all` global list should just be treated like a "policy". If we require one OR the other, then the user will not be able to select both global and specific list of policies.

Reply:
Hmm, what would be the use case for setting both? It will end up being global if "global" is assigned so I'm not sure I understand what would be the effect of separate policies in that case?

Reply:
I also don't see why a Trusted App would need to be both global and for specific polices.  Seems like it'd be one or the other.

Reply:
Disregard my comment. It was (I think) my missunderstanding. I think you are both correct: a TA entry can have 3 states of association to policies: a) none and NOT global; b) global; c: policy specific